File: iniciante/javascript/1047.js
var input = require('fs').readFileSync('stdin', 'utf8')
var lines = input.split('\n')

function relogio(a, b, c, d){
    let horaInicio = a
    let minInicio = b
    let horaFinal = c
    let minFinal = d
    let countHora = 0
    let countMinuto = 0

    while(true){
        minInicio++
        countMinuto++

        if(countMinuto == 60){
            countHora++
            countMinuto = 0
        }

        if(minInicio == minFinal){
            break
        }

        if(minInicio == 59){
            minInicio = -1
            horaInicio++
        }

    }
    

    while(true){
        if(horaInicio == horaFinal){
            break
        }

        horaInicio++
        countHora++
        
        if(horaInicio == 24){
            horaInicio = 0
        }

    }

   
    return `O JOGO DUROU ${countHora} HORA(S) E ${countMinuto} MINUTO(S)`
}

let entrada = lines.shift().split(' ')
let a = parseInt(entrada[0])
let b = parseInt(entrada[1])
let c = parseInt(entrada[2])
let d = parseInt(entrada[3])

console.log(relogio(a, b, c, d))

File: iniciante/python/1047.py
# -*- coding: utf-8 -*-

entrada = input().split(' ')
entrada = [int(x) for x  in entrada]

hora_inic, minuto_inic, hora_final, minuto_final = entrada

def calcular_hora(a, b, c, d):
    count_horas = 0
    count_minutos = 0

    while True:
        b += 1
        count_minutos += 1

        if count_minutos == 60:
            count_horas += 1
            count_minutos = 0

        if a == c and b == d:
            break

        if b == 60:
            a += 1

        if a == c and b == d:
            break

        if b == 60:
            b = 0
        
        if a == c and b == d:
            break

        if a == 24:
            a = 0
        
        if a == c and b == d:
            break

    return count_horas, count_minutos


hora, minuto = calcular_hora(hora_inic, minuto_inic, hora_final, minuto_final)
print(f'O JOGO DUROU {hora} HORA(S) E {minuto} MINUTO(S)')
